EPFParser.py

```diff
@@ -200,10 +206,11 @@ def nextRowString(self, ignoreComments=True):
         lst = []
         isFirstLine = True
         while (True):
-            ln = self.eFile.readline().decode("utf-8")
-            if (not ln): #end of file
+            ln = self.eFile.readline()
+            if (not ln or len(ln) == 0 or ln[0] == "\x00"): #end of file - skipping zero-fill at the end of tarfile
```

Comment:
Potentially we can use `line.rstrip('\x00')` to clean the line here?

Reply:
I'd rather not do that on every line; basically this is a hack for the fact that the last line of the tarfile is zero-filled to pad it to 512 bytes. But also, nulls never appear in Apple's files, so it's a safe check.